make_os/build/make_rootfs.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+	echo "Error: Number of argument should be 1"
+	exit 1 
+fi
+
+DEBOOTSTRAP_ARCH=$1
+
+if [ ${DEBOOTSTRAP_ARCH} = "armhf" ]; then
```

Comment:
Please use `"` in test commands to avoid syntax error in the case of variable ${DEBOOTSTRAP_ARCH} is brank.
```suggestion
if [ "${DEBOOTSTRAP_ARCH}" = "armhf" ]; then
```